QUARKUS-5654.md

```diff
@@ -0,0 +1,34 @@
+# QUARKUS-5654 Update MP REST Client to 4.0
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5654
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/43959
+
+## Scope of the testing
+- New overloaded `baseUri(String uri)` method so users don’t have to call URI.create()
+- New `header(String name, Object value)` method for adding headers to Client instances
+- Clarify specification CDI support (`@RestClient` qualifier is not optional)
```

Comment:
Where is it clarified? I take it you basically checked there is a test somewhere that verifies that it injection doesn't work without `@RestClient` qualifier?

Reply:
The clarification is in MP Rest specification